### PR TITLE
[CU-30ed6d0] Updated Crowdloan docs

### PR DESCRIPTION
**Pull request by PoisonPhang**
## Issue
[[CU-30ed6d0]](https://app.clickup.com/t/30ed6d0)

## Description
Includes minor refactorings of `do_claim` and `get_remote_account` to make their returns, functionally, more obvious

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-30ed6d0 [Crowdloan Rewards] Update cargo docs & README for Blocks -> Moment](https://app.clickup.com/t/30ed6d0)

File: code/parachain/frame/crowdloan-rewards/src/lib.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ Crowdloan rewards pallet used by contributors to claim their rewards.
 A user is able to claim rewards once it has an associated account. Associating
 an account using the `associate` extrinsic automatically yield the upfront
 liquidity (% of the vested reward). The rest of the reward can be claimed every
-`VestingStep` starting at the block when the pallet has been initialized
+`VestingStep` starting at the timestamp when the pallet has been initialized
```
Comment (benluelo) [inline code/parachain/frame/crowdloan-rewards/src/lib.rs:7]:
```suggestion
`VestingStep` starting at the timestamp when the pallet was initialized
```

File: code/parachain/frame/crowdloan-rewards/src/lib.rs [hunk 2]
```diff
@@ -347,6 +351,14 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Associates a reward account with some remote account provided by a proof. Calls
+		/// `do_claim` to preform the first claim.
```
Comment (benluelo) [inline code/parachain/frame/crowdloan-rewards/src/lib.rs:355]:
```suggestion
		/// `do_claim` to perform the first claim.
```

**Review by benluelo — APPROVED** [review]

**Review by andresmechali — APPROVED** [review]
LGTM

**Pull request merged**